### DISCUSSION TITLE
Unable to connect to WebDAV

### DISCUSSION THREAD

**Discussion by Hack3rDan** [photoprism/photoprism#3607] [Q&A] [answered]
Hello! I am trying to use PhotoSync to connect to my photo prism server. I am connecting over the local network using http. I’ve attached a screenshot of the error I get.

![IMG_7248](https://github.com/photoprism/photoprism/assets/1166060/247272c5-46ba-42f4-a56b-15b9c9d7d0c5)

I get the same error when I leave the password blank.

I also tried using ssl with cloud flare tunnel and NGINX reverse proxy (separately) but I get the same error with SSL.

I don’t have any other WebDAV instances to test with so at this point I’m not sure if it’s a PhotoSync or a photo prism issue.

**Comment by lastzero** [comment]
Are you commented with your local network via wifi?

**Reply by Hack3rDan** [reply]
Yes. The example screenshot shows me connected via cloudflare warp, but I get the same result on the local network with no warp.

I also attempted to connect to WebDAV via my macOS desktop and I get a similar error.

**Reply by lastzero** [reply, marked as answer]
![Screenshot_20230805_163053](https://github.com/photoprism/photoprism/assets/301686/5cab1d11-e2d0-4e29-b5fc-f392ceb19293)

That's a screenshot of my account settings. No issues at all, but it may be that new PhotoSync versions have restrictions when it comes to using HTTP connections without encryption.

Since "bill" might not be the default super admin account you should check if WebDAV access is enabled for that user.

**Reply by Hack3rDan** [reply]
Tried with the admin account. Http and https. Tested using cloudflare, same error.

I’ve verified in the configuration that I am not in public mode and that webdav is enabled.

edit: got it working with the admin account. Also had to make sure the password was in the configuration. Also had to use the WebDAV target rather than the photo prism target. Thanks for the help!
